modules/generative-openai/config/class_settings.go

```diff
@@ -48,6 +48,7 @@ var (
 	DefaultOpenAIFrequencyPenalty = 0.0
 	DefaultOpenAIPresencePenalty  = 0.0
 	DefaultOpenAITopP             = 1.0
+	DefaultOpenAIBaseURL          = "https://api.openai.com"
```

Comment:
I think that when `"https://api.openai.com"` host will be left as a default option in clients and only will be overridden in case if someone explicitly provides a value then we can remove this `DefaultOpenAIBaseURL` const and change `BaseURL()` to return empty string if not set.
```
func (ic *classSettings) BaseURL() string {
	return *ic.getStringProperty("baseURL", "")
}
```